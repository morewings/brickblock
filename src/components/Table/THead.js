import React from 'react';
import PropTypes from 'prop-types';

const THead = props => (
  <thead>
    <tr>
      <th>#</th>
      <th>First Name</th>
      <th>Last Name</th>
      <th>Username</th>
    </tr>
  </thead>
);

export default THead;
