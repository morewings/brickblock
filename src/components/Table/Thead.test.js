import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import THead from 'components/Table/THead';

describe('components > Table > THead', () => {
  it('renders without crashing', () => {
    const wrapper = shallow(<THead />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
