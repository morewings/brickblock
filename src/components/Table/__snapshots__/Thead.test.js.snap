// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components > Table > THead renders without crashing 1`] = `
<thead>
  <tr>
    <th>
      #
    </th>
    <th>
      First Name
    </th>
    <th>
      Last Name
    </th>
    <th>
      Username
    </th>
  </tr>
</thead>
`;
